scripts/triggers.coffee

```diff
@@ -49,13 +49,13 @@ triggers =
   '!goal'         : "GOOOOOOOOOOOOOOOOOOOOOOAAAAAAAAAAAAAAAAALLLLLLLLLL!!!!!!!"
   '!success'      : "http://i.imgur.com/t11sbfh.jpg"
   '!unstoppable'  : "I am unstoppable!!! http://i.imgur.com/ALHS4Za.png"
-  '!magic'        : "http://www.reactiongifs.com/r/mgc.gif"
+  '!magic'        : "http://i.imgur.com/2DyorCH.gif"
   '!mindblown'    : "http://www.reactiongifs.com/wp-content/uploads/2011/09/mind_blown.gif"
   '!troll'        : "http://25.media.tumblr.com/tumblr_lxgcxtBVcB1qh01r8o1_500.gif"
   '!pew'          : "https://img.aebian.org/pew.jpg"
   '!doit'         : "https://www.youtube.com/watch?v=nuHfVn_cfHU"
   '!trustnobody'  : "https://img.aebian.org/Duck_Dog.jpg"
-
+  '!twig'  : "https://i.imgur.com/JGbJK7g.gif"
```

Comment:
Could you put this on the line below !trustnobody instead of removing the blank line please? It helps to separate the functional code from the trigger list.